test/test_optim.py

```diff
@@ -1291,6 +1298,55 @@ def closure():
             res2 = optim_neg_inf.step(closure)
             self.assertEqual(type(res1), type(res2))
 
+    @onlyCPU
+    def test_grad_scaling_autocast_fused_optimizers(self):
```

Comment:
May I know why it is CPU only?

Reply:
It is copied from CUDA-only UT in `test_cuda.py`. I wish to unified them after fused `SGD/AdamW` kernel is also enabled on CPU. Added TODO on it.

Reply:
Instead of TODOs, let's create an issue tracker of all the plans we have for fused cpu optims. This would be better for clean code, visibility, and tracking in general.

Reply:
Hi, @janeyx99. I have create an issue to track our plan https://github.com/pytorch/pytorch/issues/123238.
And I also add support for `AdamW` in this PR https://github.com/pytorch/pytorch/pull/123074/commits/84929f0cc3a76e069bf6f6120cb973e2be48a8ca.